### PR TITLE
Add recipe for removestar

### PR DESCRIPTION
**Pull request by asmeurer**
removestar is a new tool I wrote to automatically remove star imports in
Python file. See https://www.asmeurer.com/removestar/.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Closes https://github.com/asmeurer/removestar/issues/1

### CODE REVIEW

File: recipes/removestar/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (isuruf) [inline recipes/removestar/LICENSE]: Can you create an issue or PR upstream to include this in sdists?

**Comment by asmeurer** [comment]
It’s my package. And I already did it but I didn’t think it was worth doing
a point release just for the license file.

**Comment by isuruf** [comment]
Okay. Let me know once this PR is ready for review

**Comment by asmeurer** [comment]
It’s ready now.

**Comment by asmeurer** [comment]
Thanks.

**Pull request merged**